### PR TITLE
some minor update

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
1. support search all kind experiment
2. remove `workers` config in stress memory, because every worker will cost some cpu load, so just use one worker is OK

### CODE REVIEW

File: pkg/core/stress.go [hunk 1]
```diff
@@ -47,6 +47,13 @@ func (s *StressCommand) Validate() error {
 	return nil
 }
 
+func (s *StressCommand) CompleteDefaults() {
+	if s.Workers == 0 {
+		s.Workers = 1
+	}
+
```
Comment (Andrewmatilde) [inline pkg/core/stress.go:54]: May delete newline here.

**Review by Andrewmatilde — APPROVED** [review]
Rest LGTM

File: pkg/core/stress.go (hunk 1 above)
Comment (WangXiangUSTC) [reply pkg/core/stress.go]: done

File: cmd/attack/stress.go [hunk 2]
```diff
@@ -70,11 +71,11 @@ func NewStressMemCommand(dep fx.Option, options *core.StressCommand) *cobra.Comm
 		Short: "continuously stress virtual memory out",
 		Run: func(*cobra.Command, []string) {
 			options.Action = core.StressMemAction
+			options.CompleteDefaults()
 			utils.FxNewAppWithoutLog(dep, fx.Invoke(stressAttackF)).Run()
 		},
 	}
 
-	cmd.Flags().IntVarP(&options.Workers, "workers", "w", 1, "Workers specifies N workers to apply the stressor.")
```
Comment (cwen0) [inline cmd/attack/stress.go:77]: Why delete this flag?

Comment (WangXiangUSTC) [reply cmd/attack/stress.go:77]: because every worker will cost some cpu load（almost 100%）, so just use one worker to reduce the cpu cost

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**